### ISSUE TITLE
Unit Test Case Automation for API

### ISSUE THREAD

**Issue by iamsamirzon** [notaryproject/roadmap#35]
**Summary** 
Automated Test Cases to be executed before every release
**Intended Outcome** 
A test suite is created over time and added itertively over the lifetime of this project development
**Additional context**

**Comment by iamsamirzon** [comment]
Unit test cases for APIs are being done as APIs are being implemented.

**Comment by dtzar** [comment]
Per our conversation in previous calls, this is not until RC-2 timeline.  There are separate issue(s) here, but would have to look them up.

**Comment by dtzar** [comment]
@iamsamirzon I believe this is now complete with your scope of Unit test per: https://github.com/notaryproject/notation/issues/197

**Comment by dtzar** [comment]
Closing on call with Samir 7/28
